SPECS/cloud-init/cloud-init.spec

```diff
@@ -107,6 +110,8 @@ echo -e 'line1\nline2\nline3\ncloud-init-ca-certs.crt\n' > "${conf_file}"
 %define test_pkgs pytest-metadata unittest2 mock attrs iniconfig netifaces pyserial
 
 pip3 install --upgrade %{test_pkgs}
+# Higher versions of 'jsonschema' break the tests.
+sed -E -i 's/jsonschema/jsonschema==4.20.0/' test-requirements.txt
```

Comment:
Not supportive of this workaround. Why can't we use our distribution's jsonschema for this? Would using our distro's jsonschema incur even more changes beyond just adding a BR?

Reply:
It would, yes. We're on version 2.6.0, while the tests have been always running on 4.xx. I don't want to rock the boat with this change, just bring it back to the previous state where the tests were stable.

Reply:
FWIW my PR that uses our existing packaged python3-jsonschema passes tests just fine.

Reply:
Ah, that's great! Please check that in and I'll close this PR in a minute. Thanks, Dan!

Reply:
Wait, I see you closed it. I'll just move that change here, so we fix the source URL in one go.